File: OLSource1/v1/VS_officedev/codesnippet/CSharp/creating-an-association-between-entities_1.cs
        public static IEnumerable<SalesOrderHeader> ContactToSalesOrder(int contactID)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");

            IEnumerable<SalesOrderHeader> orderList = 
                from orders in dataContext.SalesOrderHeaders
                     where orders.ContactID == contactID
                     select orders;
            return orderList;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/creating-an-association-between-entities_1.vb
    Public Shared Function ContactToSalesOrder(ByVal contactID As Integer) As IEnumerable(Of SalesOrderHeader)
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim orderList As IEnumerable(Of SalesOrderHeader) = _
            From orders In dataContext.SalesOrderHeaders _
            Where orders.ContactID = contactID _
                  Select orders
        Return orderList

    End Function